src/components/Modals/Tasks/CreateTask.jsx

```diff
@@ -0,0 +1,187 @@
+import PropType from 'prop-types';
+import React from 'react';
+import classes from './CreateTask.module.css';
+import Button from '../../Button/Button';
+import CraftInput from '../CraftInput';
+
+import {
+  CREATE_TASK_ONCHANGE,
+  CREATE_TASK_VALIDATE_FIELDS,
+  CREATE_TASK_VALIDATE_FORM,
+  reducerFunc,
+} from './Task-helpers';
+import { TASKS, db, USERS, createElemRef } from '../../../utilities/fb-helpers';
+import debounce from '../../../utilities/debounce';
+
+const newTaskRef = createElemRef(TASKS);
+
+export default class CreateTask extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: {
+        title: '',
+        description: '',
+        startDate: '',
+        deadLine: '',
+        selectedUsers: [],
+      },
+      validator: {
+        title: false,
+        startDate: false,
+        deadLine: false,
+        selectedUsers: false,
+      },
+      usersList: [],
+      isValid: false,
+      errors: {
+        titleError: '',
+        startDateError: '',
+        deadLineError: '',
+        selectedUsersError: '',
+      },
+    };
+    this.onChange = this.onChange.bind(this);
+    this.liftUpCreateTask = this.liftUpCreateTask.bind(this);
+    this.closeModal = this.closeModal.bind(this);
+    this.validateFields = this.validateFields.bind(this);
+    this.validateForm = this.validateForm.bind(this);
+  }
+
+  componentDidMount() {
+    db.collection(USERS)
+      .get()
+      .then((querySnapshot) => {
+        const usersList = [];
+        querySnapshot.forEach((doc) => {
+          usersList.push(doc.data());
+        });
+
+        return usersList;
+      })
+      .then((usersList) =>
+        this.setState((prevState) => ({
+          ...prevState,
+          data: {
+            ...prevState.data,
+            id: newTaskRef.id,
+          },
+          usersList,
+        })),
+      )
+      .catch((error) => {
+        console.log('Error reading users collection: ', error);
+      });
+  }
+
+  onChange(event) {
+    const { target } = event;
+    const { name, value } = target;
```

Comment:
do `spread in 1 row` if you can

Reply:
Dont in 1 row. Thx linter :D
```suggestion
    const {
      target,
      target: { name, value },
    } = event;
```